invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/utils.js

```diff
@@ -108,7 +106,7 @@ export function SearchItemCreators({ creators }) {
     return link;
   }
   return creators.map((creator, index) => (
-    <span className="creatibutor-wrap separated" key={index}>
+    <span className={spanClass} key={index}>
```

Comment:
```suggestion
    <span className={`creatibutor-wrap separated ${className}`} key={index}>
```
Can we keep "creatibutor-wrap separated" always by default, then we don't need to add these classes to the mobile view.

Reply:
I'm not sure if it would be best to allow the component to have all it's className overridden, as it would make it more reusable. I've made the change but I would like if others in the team give their opinion.

Reply:
if they should always have this class and will otherwise break the component ui then it makes sense to always pass it, otherwise i would prefer to make it completely overridable

Reply:
As far as I know, the classes belong to the component and should not be overridden. It is the name with connected identifier icon which we want to keep consistent in styling across the entire site.